sdk/core/Azure.Core.Amqp/src/AmqpMessageBody.cs

```diff
@@ -35,6 +45,30 @@ public AmqpMessageBody(IEnumerable<ReadOnlyMemory<byte>> data)
             BodyType = AmqpMessageBodyType.Data;
         }
 
+        /// <summary>
+        /// Initializes a new <see cref="AmqpMessageBody"/> instance with the
+        /// passed in value.
+        /// </summary>
+        /// <param name="value">The data sections.</param>
```

Comment:
Should we consider a `<seealso>` here that calls out to the AMQP spec to offer context around what the types and how they may be treated/encoded?